privly.safariextension/scripts/background_scripts/first_run.js

```diff
@@ -47,6 +47,7 @@ var firstRun = {
       ls.setItem("glyph_cells", glyph_cells);
     }
 
+    /* istanbul ignore if */
     if (typeof safari !== "undefined" && safari.application !== undefined) {
```

Comment:
It is easier to have the ignore comment before the if conditions than having a helper function in this case. This is because, the safari variable is used at many different places to perform different functions.